src/Dfe.PlanTech.Domain/Content/Models/Page.cs

```diff
@@ -0,0 +1,14 @@
+using Dfe.PlanTech.Domain.Content.Interfaces;
+
+namespace Dfe.PlanTech.Domain.Content.Models;
+
+public class Page : ContentComponent
+{
+    public string InternalName { get; init; } = null!;
+
+    public string Slug { get; init; } = null!;
+
+    public Title? Title { get; init; }
```

Comment:
Do we use this for populating the title in the generated html for the <head> - If so, It shouldn't be nullable. Title-less heads will cause accessibility issues